Manual.md

```diff
@@ -1302,6 +1303,36 @@ cache during the install/removal of the package
 - `font_dirs`: which should be set to the directory where the package
 installs its fonts
 
+<a id="pkg_remove"></a>
+### Removing a package
+
+Follows a list of things that should be done to help guarantee that a
+package template removal and by extension it's binary packages from
+Void Linux repositories goes smoothly.
+
+Before removing a package template:
+
+- Guarantee that no package depends on it or any of it's subpackages.
+For that you can search the templates for references to the package
+`grep -r 'depends=.*\<pkg\>' srcpkgs/` should cover it.
```

Comment:
Also I would recommend `git grep` over `grep` since it is vastly faster. I personally use `git grep pkg srcpkgs/*/template` always when I want to find reverse deps.

Reply:
Nope, grep doesn't deal with multilines

Reply:
updated to just do `grep -r pkg srcpkgs/`